cookbooks/fb_users/resources/default.rb

```diff
@@ -120,6 +122,15 @@
       password pass if pass
       action :create
     end
+
+    if manage_homedir
+      directory homedir do
+        owner mapinfo['uid']
+        group ::FB::Users::GID_MAP[pgroup]['gid'].to_i
+        mode homedir_mode if homedir_mode
```

Comment:
Not everyone's homedir will necessary be their primary group. For example my primary group could be `os` but the company could want all homedirs to be group-owned by `pe`, or vise-versa.

Since `manage_homedir` defaults to false, this probably OK. But something to think about. I would consider maybe a `homedir_group` config, then fall back to user's GID if not set.

Reply:
I like the `homedir_group` idea here